tests/e2e/test_rosacli_iam_roles.go

```diff
@@ -874,4 +874,130 @@ var _ = Describe("Edit IAM",
 				foundUserRole = userRoleList.UserRole(userrolePrefix, ocmAccountUsername)
 				Expect(foundUserRole.Linded).To(Equal("No"))
 			})
+
+		It("create/delete hypershift account roles with managed policies - [id:61322]",
+			labels.Critical,
+			func() {
+				defer func() {
+					By("Cleanup created account-roles in the test case")
+					if len(accountRolePrefixesNeedCleanup) > 0 {
+						for _, v := range accountRolePrefixesNeedCleanup {
+							_, err := ocmResourceService.DeleteAccountRole("--mode", "auto",
+								"--prefix", v,
+								"-y")
+
+							Expect(err).To(BeNil())
+						}
+					}
+				}()
+
+				var (
+					rolePrefixStable    = "prefixS"
+					rolePrefixCandidate = "prefixC"
+					rolePrefixClassic   = "prefixClassic"
+					versionStable       = "4.13"
+					versionCandidate    = "4.14"
+					path                = "/fd/sd/"
+				)
+
+				By("Get the AWS Account Id")
+				rosaClient.Runner.JsonFormat()
+
+				whoamiOutput, err := ocmResourceService.Whoami()
+				Expect(err).To(BeNil())
+				rosaClient.Runner.UnsetFormat()
+				whoamiData := ocmResourceService.ReflectAccountsInfo(whoamiOutput)
+				AWSAccountID := whoamiData.AWSAccountID
+
+				By("Create account-roles of hosted-cp in stable channel")
+				output, err := ocmResourceService.CreateAccountRole("--mode", "auto",
```

Comment:
I think the case should create account role via "rosa create account-roles".
@yuwang-RH Would you help double confirm it?

Reply:
@yingzhanredhat But It is using rosa create account-roles command only, can you please check the logs once ? Thanks